Accessing resistance report link from sample page triggers an internal server error

**Describe the bug**

Accessing the resistance report link from the /samples/ view triggers a 500 error in the client:

```
[2023-10-23 12:45:31 +0000] [8] [ERROR] Exception on /samples/11MT0500103/resistance_report [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/flask/app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/flask/app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/flask/app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/flask/app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/flask_login/utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/worker/app/app/blueprints/sample/views.py", line 289, in resistance_report
    return render_template(
           ^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/flask/templating.py", line 151, in render_template
    return _render(app, template, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/flask/templating.py", line 132, in _render
    rv = template.render(context)
         ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/jinja2/environment.py", line 1301, in render
    self.environment.handle_exception()
  File "/usr/local/lib/python3.11/site-packages/jinja2/environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/home/worker/app/app/blueprints/sample/templates/resistance_report.html", line 3, in top-level template code
    {% from "cards.html" import resistance_card, virulence_card %}
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/worker/app/app/templates/layout.html", line 2, in top-level template code
    {% from "navbar.html" import navbar %}
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/worker/app/app/templates/bootstrap_base.html", line 56, in top-level template code
    {% block body %}
  File "/home/worker/app/app/templates/bootstrap_base.html", line 57, in block 'body'
    {% block content %}
  File "/home/worker/app/app/blueprints/sample/templates/resistance_report.html", line 15, in block 'content'
    {{ sidebar(sample, navigation=False) }}
    ^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: macro 'sidebar' takes no keyword argument 'navigation'
```

**To Reproduce**

Open http://mtlucmds1.lund.skane.se/bonsai/samples/11MT0500103/resistance_report

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -4,6 +4,8 @@
 
 ### Fixed
 
+ - Resistance_report now render work in progress page
+
 ### Changed
 
 ## [v0.1.0]

--- a/frontend/app/blueprints/sample/templates/cards.html
+++ b/frontend/app/blueprints/sample/templates/cards.html
@@ -9,6 +9,49 @@
 <a class="link-dark link-underline-success" href="https://{{base_url}}/{{db.version}}" target="_blank">{{ db.version[:6] }}</a>
 {% endmacro %}
 
+
+{% macro sample_header(sample) %}
+<div class="br-header p-2">
+    <div class="card">
+        <div class="card-body">
+            <div class="d-flex">
+                <h4 id="sample-id-heading" class="card-title">{{sample.sample_id}}</h4>
+                {% for tag in sample.tags %}
+                    {% if tag.type == "virulence" %}
+                        {% set heading="virulence-heading" %}
+                    {% else %}
+                        {% set heading="phenotype-heading" %}
+                    {% endif %}
+                    <span 
+                        class="tag badge ms-2 rounded-pill bg-{{tag.severity}}"
+                        data-bs-toggle="tooltip" data-bs-placement="top"
+                        {% if tag.description %}data-bs-title="{{ tag.description }}"{% endif %}
+                    >
+                        {{tag.label | upper }}
+                    </span>
+                {% endfor %}
+            </div>
+            <div class="row gx-5">
+                <div class="col-md-auto">
+                    Upload date: {{sample.created_at | strftime}}
+                    <br>
+                    <p class="card-text">
+                        {% if sample.qc_status.status == "passed" %}
+                            QC: <strong class="text-success">{{ sample.qc_status.status | capitalize }}</strong>
+                        {% elif sample.qc_status.status == "failed" %}
+                            QC: <strong class="text-danger">{{ sample.qc_status.status | capitalize }}</strong>; {{ sample.qc_status.action }}, {{ sample.qc_status.comment }}
+                        {% else %}
+                            QC: <strong>{{ sample.qc_status.status | capitalize }}</strong>
+                        {% endif %}
+                    </p>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+{% endmacro %}
+
+
 {% macro accession_link(accession, name=None) %}
     <a class="text-dark" target="_blank" href="https://www.ncbi.nlm.nih.gov/protein/{{accession}}">
         {% if name == None %}

--- a/frontend/app/blueprints/sample/templates/resistance_report.html
+++ b/frontend/app/blueprints/sample/templates/resistance_report.html
@@ -1,6 +1,6 @@
 {% extends "layout.html" %}
-{% from "sidebar.html" import sidebar %}
-{% from "cards.html" import resistance_card, virulence_card %}
+{% from "sidebar.html" import sidebar, table_of_content %}
+{% from "cards.html" import sample_header, resistance_card, virulence_card %}
 
 {% block css %}
 {{ super() }}
@@ -10,28 +10,19 @@
 
 {% block content %}
 {% set footnotes = [] %}
-<div class="container-fluid">
-    <div class="row">
-        {{ sidebar(sample, navigation=False) }}
-        <div class="col-md-9 ms-sm-auto col-lg-10 bg-white px-md-4 pb-4 container" data-bs-spy="scroll" data-bs-target="#sidebar">
-            <div id="sample-id-heading" class="pt-4">
-                <h1>Resistance report: {{sample.sampleId}}</h1>
-            </div>
-            <h4>Overview</h4>
-            <hr>
-            <div id="phenotype-heading" class="row justify-content-md-center">
-                <div class="col-md-auto">
-                {% for res in sample.phenotypeResult %}
-                    {% if res.type == "antimicrobial_resistance" %}
-                        <div class="col-md-auto">
-                        {{ resistance_card(res) }}
-                        </div>
-                    {% endif %}
-                {% endfor %}
+<div class="container-fluid br-layout">
+    {{ sidebar(sample) }}
+    <main class="br-main order-1">
+        {{ sample_header(sample) }}
+        {{ table_of_content() }}
+        <div class="container-fluid">
+            <div class="row justify-content-md-center">
+                <div class="card col-sm-12 col-md-auto">
+                    <h4>Work in progress...</h4>
                 </div>
             </div>
         </div>
-    </div>
+    </main>
 </div>
 <script>
     const tooltipTriggerList = document.querySelectorAll('[data-bs-toggle="tooltip"]')

--- a/frontend/app/blueprints/sample/templates/sample.html
+++ b/frontend/app/blueprints/sample/templates/sample.html
@@ -1,6 +1,6 @@
 {% extends "layout.html" %}
 {% from "sidebar.html" import sidebar, table_of_content, qc_form_controls_js %}
-{% from "cards.html" import resistance_summary_card, resistance_table_card, virulence_card, species_prediction_card, amr_prediction_card, stx_card, analysis_meta_pipeline, analysis_meta_databases, mlst_card, similar_samples_card, qc_card_postalign, qc_card_generic, cgmlst_qc_card %}
+{% from "cards.html" import sample_header, resistance_summary_card, resistance_table_card, virulence_card, species_prediction_card, amr_prediction_card, stx_card, analysis_meta_pipeline, analysis_meta_databases, mlst_card, similar_samples_card, qc_card_postalign, qc_card_generic, cgmlst_qc_card %}
 
 {% block css %}
 {{ super() }}
@@ -14,46 +14,6 @@
 <script src="https://unpkg.com/tidytree@0.5.9/dist/tidytree.min.js"></script>
 {% endblock scripts %}
 
-{% macro sample_header(sample) %}
-<div class="br-header p-2">
-    <div class="card">
-        <div class="card-body">
-            <div class="d-flex">
-                <h4 id="sample-id-heading" class="card-title">{{sample.sample_id}}</h4>
-                {% for tag in sample.tags %}
-                    {% if tag.type == "virulence" %}
-                        {% set heading="virulence-heading" %}
-                    {% else %}
-                        {% set heading="phenotype-heading" %}
-                    {% endif %}
-                    <span 
-                        class="tag badge ms-2 rounded-pill bg-{{tag.severity}}"
-                        data-bs-toggle="tooltip" data-bs-placement="top"
-                        {% if tag.description %}data-bs-title="{{ tag.description }}"{% endif %}
-                    >
-                        {{tag.label | upper }}
-                    </span>
-                {% endfor %}
-            </div>
-            <div class="row gx-5">
-                <div class="col-md-auto">
-                    Upload date: {{sample.created_at | strftime}}
-                    <br>
-                    <p class="card-text">
-                        {% if sample.qc_status.status == "passed" %}
-                            QC: <strong class="text-success">{{ sample.qc_status.status | capitalize }}</strong>
-                        {% elif sample.qc_status.status == "failed" %}
-                            QC: <strong class="text-danger">{{ sample.qc_status.status | capitalize }}</strong>; {{ sample.qc_status.action }}, {{ sample.qc_status.comment }}
-                        {% else %}
-                            QC: <strong>{{ sample.qc_status.status | capitalize }}</strong>
-                        {% endif %}
-                    </p>
-                </div>
-            </div>
-        </div>
-    </div>
-</div>
-{% endmacro %}
 
 {% block content %}
 {% set footnotes = [] %}

